localstack-core/localstack/aws/protocol/op_router.py

```diff
@@ -8,14 +8,14 @@
 from werkzeug.routing import Map, MapAdapter
 
 from localstack.aws.protocol.routing import (
-    GreedyPathConverter,
     StrictMethodRule,
     path_param_regex,
     post_process_arg_name,
     transform_path_params_to_rule_vars,
 )
 from localstack.http import Request
 from localstack.http.request import get_raw_path
+from localstack.services.edge import GreedyPathConverter
```

Comment:
i know i said "just add the greedy match router + converter to edge.py?", but really meant either copy&pasting it here, or move it to a more generic place and import it from there :-)

i don't like the `localstack.services.edge` import here.

i'd suggest moving the `GreedyPathConverter` to `localstack.http.router` for now, and perhaps also introduce it in rolo.

Reply:
This is much nicer, thanks 👌 I didn't like it either, it felt very out of place. This looks better!